Added a missing include file (necessary for gid_t and others).
So far as I am aware, this has not been added upstream yet.
--- CSSC-1.3.0/src/file.h	2010-05-16 19:31:33.000000000 +0200
+++ CSSC-1.3.0/src/file.h	2014-02-03 21:48:30.000000000 +0100
@@ -30,6 +30,7 @@
 #ifndef CSSC__FILE_H__
 #define CSSC__FILE_H__
 
+#include <sys/types.h>
 #include "filelock.h"
 
 enum create_mode {
